Decisions/scs-xxxx-v1-ssd-flavors.md

```diff
@@ -0,0 +1,249 @@
+title: SSD Flavors
+type: Decision Record
+status: Draft
+track: IaaS
+version: v0.1
+enhances: scs-xxxx-v1-flavor-naming.md
+
+# Introduction
+
+SCS defines an IaaS Flavor Naming standard that mandates a number of standard flavors
+to be available in each SCS-compliant IaaS offering. While offering or exposing
+IaaS is not a requirement for SCS-compliant infrastructure offerings -- SCS allows
+for platforms only exposing the container layer (plus S3 compatible object storage)
+for gen2 (container-based) cloud-native workloads --
+the SCS reference implementation does include a complete IaaS implementation that
+many providers want to expose as they have customers desiring access at this layer
+for gen1 (VM-based) cloud-native workloads or for the virtualization of more
+classical (not cloud-native) workloads. The IaaS implementation thus comes with
+standards.
+
+This Decision Record is about adding a few mandatory flavors on the IaaS level
+that include flavors with local SSD (or better) storage.
+
+# Motivation
+
+The [currently defined standard flavors](https://github.com/SovereignCloudStack/Docs/blob/main/Design-Docs/flavor-naming.md)
+(as of v1.1 from 2022-09-08) do not include
+flavors that use local storage. For certain workloads such as big data (Hadoop),
```

Comment:
I'd refrain from naming explicit products, if not really needed.

Reply:
Ack. I had the had hdfs in mind where a networked disk is a particularly bad match. I change the wording to "big data filesystem."